src/nnbench/core.py

```diff
@@ -178,7 +178,14 @@ def decorator(fn: Callable) -> list[Benchmark]:
                 )
             names.add(name)
 
-            bm = Benchmark(fn, name=name, params=params, setUp=setUp, tearDown=tearDown, tags=tags)
+            bm = Benchmark(
```

Comment:
Did ruff change this formatting?

Reply:
Ah, yes. The linebreak was introduced when i added the state. Apparently ruff did not remove the linebreak even after I removed the passing of `State`.